src/wp-includes/theme.php

```diff
@@ -3232,6 +3232,15 @@ function get_registered_theme_feature( $feature ) {
 	return isset( $_wp_registered_theme_features[ $feature ] ) ? $_wp_registered_theme_features[ $feature ] : null;
 }
 
+/**
+ * Returns whether the current theme supports block templates
+ *
+ * @return boolean Whether the current theme supports block templates.
+ */
+function theme_supports_block_templates() {
+	return current_theme_supports( 'block-templates' );
```

Comment:
How about just inlining this? The check is super simple now; if we need more criteria in the future, we can always add them later, but maybe it's best to start simple (YAGNI!)

Reply:
Inlining in e4ecc6b390. My rationale is that we don't add more API surface (even if only internal) for now; and in some places, we might need a slightly different check anyway later, so it might make sense to keep individual checks separate. (Feel free to revert tho.)